codegen/module_system.go

```diff
@@ -690,6 +691,16 @@ func (g *EndpointGenerator) generateEndpointFile(
 			strings.Title(method.Name) + "Endpoint"
 	}
 
+	// TODO: http client needs to support multiple thrift services
+	var clientMethodName string
+	if method.DownstreamMethod != nil {
```

Comment:
Can `method.DownstreamMethod` be `nil` here? (i don't think it can)

If it can, an error should be thrown here. If it can't, this `if` should be removed